MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m

```diff
@@ -536,6 +536,9 @@ - (MUXSDKPlayerData *)getPlayerData {
         }
     }
     // TODO: Airplay - don't set the view if we don't actually know what is going on.
+    if (_player.externalPlaybackActive) {
```

Comment:
I tried testing Airplay detection using https://www.airserver.com/Mac, but I'm always getting false for AVPlayer's `externalPlaybackActive` property. :disappointed: I don't have an actual Apple TV but I would like to test it on an actual device to see if I get the same behavior or if this is just a quirk of AirServer

Reply:
Will test this for you like we discussed